### ISSUE TITLE
Highlighting issues with abi.* commands

### ISSUE THREAD

**Issue by eMarchenko** [davidhq/SublimeEthereum#38]
```
pragma solidity 0.5.4;
pragma experimental ABIEncoderV2;

contract C {
	function() external {
		abi.encode(1);
		abi.decode(msg.data, (bytes4, uint, uint));
	}
}
```
looks incorrect for me:
![screenshot 6](https://user-images.githubusercontent.com/7409234/53670585-4ad3e700-3c8c-11e9-9508-6ee7c040e2a4.png)
I guess it's because of 
* special status of `abi` object
* and unique turple of types in `decode`

**Comment by davidhq** [comment]
looking into this today or next few days

meanwhile please report if previous changes have broken something... 

I identified some other smaller issues too, will try to fix as well... because of multiple small fixes, I really appreciate fast feedback if anything should break (I really hope not ^_ )

**Comment by davidhq** [comment]
should be fixed in `v1.1.15`

**Comment by davidhq** [comment]
<img width="329" alt="screen shot 2019-03-07 at 09 55 34" src="https://user-images.githubusercontent.com/3899/53944206-2da17d00-40bf-11e9-991c-2eded99b64e6.png">
